packages/test-scenarios/__tests__/scenario-test.js

```diff
@@ -0,0 +1,174 @@
+const { Scenarios, Project } = require('scenario-tester');
+const { dirname, delimiter } = require('path');
+const { merge } = require('lodash');
+
+jest.setTimeout(500000);
+
+// https://github.com/volta-cli/volta/issues/702
+// We need this because we're launching node in child processes and we want
+// those children to respect volta config per project.
+(function restoreVoltaEnvironment() {
+  let voltaHome = process.env['VOLTA_HOME'];
+  if (!voltaHome) return;
+  let paths = process.env['PATH'].split(delimiter);
+  while (/\.volta/.test(paths[0])) {
+    paths.shift();
+  }
+  paths.unshift(`${voltaHome}/bin`);
+  process.env['PATH'] = paths.join(delimiter);
+})();
+
+// eslint-disable-next-line node/no-unpublished-require
+const babelPluginPath = require.resolve('../../babel-plugin-ember-test-metadata/dist/index');
```

Comment:
let's move this into a `link:` dep of `packages/test-scenarios/package.json` and then inline in the `ember-cli-build.js` we can do `babel: { plugins: [ require.resolve('babel-plugin-ember-test-metadata') ] }`.

Reply:
Ya let's do that